couchbase-sys/Cargo.toml

```diff
@@ -21,7 +21,7 @@ cc = "1.0"
 cmake = "0.1.45"
 
 [build-dependencies.bindgen]
-version = "0.55"
+version = "0.62"
```

Comment:
This should probably be an update to the latest version.

Reply:
I agree